interventions/models.py

```diff
@@ -14,4 +14,11 @@ class Operation(models.Model):
     date = models.DateField()
     time = models.TimeField()
 
+class Patient(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    disease = models.CharField(max_length=128)
```

Comment:
Przemianowałbym to na hospitalization_case (jako przyczyna hospitalizacji, disease znaczy bardziej choroba, zaraza. Niekoniecznie sprawdzi się w przypadku np. złamania)